src/Entity/Contract.php

```diff
@@ -18,9 +19,11 @@ class Contract
     private ?string $name = null;
 
     #[ORM\Column]
+    #[Assert\Positive]
```

Comment:
Does Positive support 0 ? if it's not, we must have this feature because in case the user has free contract